frontend/src/components/VTag/VTag.vue

```diff
@@ -4,8 +4,9 @@
     size="small"
     variant="filled-gray"
     class="label-bold"
-    :href="href"
-    >{{ title }}</VButton
+    v-bind ="$props"
```

Comment:
`props` are automatically passed to the child component in Vue, but some `attrs` are not. We need to bind `attrs` here.
```suggestion
    v-bind ="$attrs"
```

Reply:
I am going to draft this PR until the changes are addressed (linting and binding $attrs instead of $props)